app/models/step.rb

```diff
@@ -10,6 +10,7 @@ class Step < ActiveRecord::Base
 
   belongs_to :user
   belongs_to :proposal
+  belongs_to :completed_by, class_name: "User"
```

Comment:
what about `belongs_to :completer` ? That seems more consistent with the other AR relationships we have in this class and elsewhere in the project